src/test/java/com/odeyalo/sonata/connect/repository/storage/RepositoryDelegatePlayerStateStorageTest.java

```diff
@@ -17,13 +20,23 @@ static class PersistablePlayerStateTestEntityGenerator implements TestEntityGene
         public PersistablePlayerState generateValidEntity() {
             return PersistablePlayerState.builder()
                     .id(1L)
+                    .repeatState(RepeatState.OFF)
+                    .shuffleState(false)
+                    .playing(true)
+                    .playingType(PlayingType.TRACK)
+                    .user(InMemoryUserEntity.builder().id("mikuuuu").build())
                     .build();
         }
 
         @Override
         public PersistablePlayerState generateInvalidEntity() {
             return PersistablePlayerState.builder()
```

Comment:
I  should create the PersistablePlayerStateFaker class to re-use logic to create PersistablePlayerState